electron-app/src/main.ts

```diff
@@ -27,6 +28,9 @@ app.commandLine.appendSwitch('disable-renderer-backgrounding');
 app.commandLine.appendSwitch('disable-background-timer-throttling');
 app.commandLine.appendSwitch('disable-features', 'CrossOriginOpenerPolicy');
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageJson = require('../../package.json');
```

Comment:
nit: use app.getVersion()
Not sure how stable getting the direct file is when packaged.

Reply:
@mvdicarlo Attempting that gave me a wildly different version number:

```
PostyBirb/19.1.9 (by mvdicarlo on GitHub) Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.167 Electron/19.1.9 Safari/537.36
```

No idea where that comes from.

Reply:
It's the electton version.
When bundled it is the package json version.